pages/k8s/charm-kubernetes-master.md

```diff
@@ -0,0 +1,629 @@
+---
+wrapper_template: "kubernetes/docs/base_docs.html"
+markdown_includes:
+  nav: "kubernetes/docs/shared/_side-navigation.md"
+context:
+  title: "kubernetes-master charm"
+  description: Kubernetes-master Charm reference
+keywords: kubernetes-master, charm, config
+tags: [reference]
+sidebar: k8smain-sidebar
+permalink: charm-kubernetes-master.html
+layout: [base, ubuntu-com]
+toc: False
+---
+[Kubernetes](http://kubernetes.io/) is an open source system for managing
```

Comment:
Drop these first 2 intro paragraphs, which are covered in our Home/Overview docs pages.